test/server/cardImplementations/01_SOR/MonMothmaVoiceoftheRebellion.spec.js

```diff
@@ -0,0 +1,120 @@
+describe('Mon Mothma, Voice of the Rebellion', function() {
+    integration(function() {
+        describe('Mon Mothma\'s Ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['mon-mothma#voice-of-the-rebellion'],
+                        deck: ['cell-block-guard', 'pyke-sentinel', 'volunteer-soldier', 'cartel-spacer', 'battlefield-marine', 'wampa', 'viper-probe-droid', 'snowtrooper-lieutenant'],
+                        deckSize: 8,
+                        leader: ['leia-organa#alliance-general'],
+                        resources: ['atst', 'atst', 'atst', 'atst', 'atst', 'atst']
+                    }
+                });
+
+                this.monMothma = this.player1.findCardByName('mon-mothma#voice-of-the-rebellion');
+                this.battlefieldMarine = this.player1.findCardByName('battlefield-marine');
+
+                this.cartelSpacer = this.player1.findCardByName('cartel-spacer');
+                this.cellBlockGuard = this.player1.findCardByName('cell-block-guard');
+                this.pykeSentinel = this.player1.findCardByName('pyke-sentinel');
+                this.volunteerSoldier = this.player1.findCardByName('volunteer-soldier');
+
+                this.noMoreActions();
+            });
```

Comment:
Tests look great overall, very thorough. Couple edge cases that might be worth adding:
- Deck has less than five cards remaining
- Deck has no cards remaining